charts/metal3/0.3.0/charts/baremetal-operator/crds/customresource-bmceventsubscriptions.yaml

```diff
@@ -0,0 +1,85 @@
+---
```

Comment:
Not related to this PR but noting it here before I forget - we should perhaps consider moving the CRDs into a separate chart similar to `metallb-crds` - these will change sometimes on BMO upgrade so we'll need a way to handle that, and it will reduce the duplication between metal3 chart versions